hyco-https/lib/HybridConnectionHttpsServer.js

```diff
@@ -317,6 +318,9 @@ function connectControlChannel(server) {
   var opt = null;
   var token = null;
   var tokenRenewDuration = null;
+  var reconnectDelays = [1, 2, 5, 10, 30]; // in seconds
```

Comment:
>1 [](start = 25, length = 1)

If the listener had been connected successfully, then gets dropped for some reason, does the first attempt wait 1 second? I personally don't think the first one should delay any.

Reply:
Would putting a 0 at the start of this list do what I'm talking about?

---
In reply to: [316927208](https://github.com/Azure/azure-relay-node/pull/35#discussion_r316927208) [](ancestors = 316927208)

Reply:
beginning with 0 makes sense

---
In reply to: [316928779](https://github.com/Azure/azure-relay-node/pull/35#discussion_r316928779) [](ancestors = 316928779,316927208)